wallet/src/de/schildbach/wallet/ui/dashpay/ContactsFragment.kt

```diff
@@ -83,6 +79,9 @@ class ContactsFragment : BottomNavFragment(R.layout.fragment_contacts_root), Tex
 
     override val navigationItemId = R.id.contacts
 
+    private var _binding: FragmentContactsRootBinding? = null
+    private val binding get() = _binding!!
```

Comment:
Small fix for ContactsFragment: replaced lateinit bindings with nullable bindings. Lateinit bindings [aren't recommended](https://developer.android.com/topic/libraries/view-binding#fragments) for fragments since fragments outlive their views and a memory leak can be introduced if binding isn't nullified in `onDestroyView()`